rfc8446.xml

```diff
@@ -109,84 +107,120 @@ compatible with previous versions, all versions of TLS incorporate a
 versioning mechanism which allows clients and servers to interoperably
 negotiate a common version if one is supported by both peers.</t>
 
-<t>This document supersedes and obsoletes previous versions of TLS
-including version 1.2 <xref target="RFC5246"/>.  It also obsoletes the TLS ticket
-mechanism defined in <xref target="RFC5077"/> and replaces it with the mechanism
-defined in <xref target="resumption-and-psk"/>. <xref target="negotiated-groups"/> updates
-<xref target="RFC4492"/> by modifying the protocol attributes used to negotiate
-Elliptic Curves.  Because TLS 1.3 changes the way keys are derived, it
-updates <xref target="RFC5705"/> as described in <xref target="exporters"/>.  It also changes
-how OCSP messages are carried and therefore updates <xref target="RFC6066"/>
-and obsoletes <xref target="RFC6961"/> as described in section <xref target="ocsp-and-sct"/>.</t>
+<t>This document supersedes and obsoletes previous versions of TLS,
+including version 1.2 <xref target="RFC5246"/>.  It also obsoletes the TLS
+ticket mechanism defined in <xref target="RFC5077"/> and replaces it with the
+mechanism defined in <xref target="resumption-and-psk"/>.
+
+<!-- [rfced] Sections 1 and 4.6.1:  Please confirm that
+Erratum #4800 for RFC 5077 <https://www.rfc-editor.org/errata/eid4800>>
+is addressed by both sentences below.  (We ask this because we wonder
+if the sentence from Section 4.6.1 should be updated to state that
+the mechanism from RFC 5077 has been replaced.)
```

Comment:
A couple of point 1) the text in this section needs to explain why we've got 5077 listed in the obsoletes section so I'm not sure it's really good to remove it, and; 2) you are right that we do not fix the 5077 construction, but would a note here really hurt?  Is there some untested code path that will result in a problem when a ticket bigger than 2^16-1 in passed to TLS?

Reply:
Please let us know how/if the text should be updated.

Reply:
I’m not sure I see why we need to say anything about the construction.  From what I remember it might be good to obsolete it. It should not affect interoperability.

Reply:
It definitely does not affect interoperability, so no objection to staying mum about it.